test/middleware/PenaltyReferenceRouter.test.ts

```diff
@@ -0,0 +1,184 @@
+import { Penalty, PenaltyList } from 'ch-sdk-node/dist/services/lfp';
+import { assert, expect } from 'chai';
+import { NextFunction, Request, Response } from 'express';
+import { createSubstituteOf } from '../SubstituteFactory';
+import { createSession } from '../utils/session/SessionFactory';
+
+import { PenaltyReferenceRouter } from 'app/middleware/PenaltyReferenceRouter';
+import { ApplicationData, APPLICATION_DATA_KEY } from 'app/models/ApplicationData';
+import { Reasons } from 'app/models/Reasons';
+import { APPLICATION_DATA_UNDEFINED, SESSION_NOT_FOUND_ERROR } from 'app/utils/CommonErrors';
+import { REVIEW_PENALTY_PAGE_URI } from 'app/utils/Paths';
+
+
```

Comment:
No need for this large gap between tests and imports, also I noticed that you havnt put empty lines between tests, probably worth doing so.